test/controllers/pages_controller_test.rb

```diff
@@ -0,0 +1,24 @@
+require 'test_helper'
+
+class PagesControllerTest < ActionDispatch::IntegrationTest
+  test "should get index" do
+    get pages_index_url
+    assert_response :success
+  end
+
+  test "should get about" do
+    get pages_about_url
+    assert_response :success
+  end
+
+  test "should get meetup" do
+    get pages_meetup_url
+    assert_response :success
+  end
+
+  test "should get contact" do
+    get pages_contact_url
+    assert_response :success
+  end
+
+end
```

Comment:
Cool auto generated tests huh?

If you run `rails t` in your console it will run all the tests for you. Give it a try!